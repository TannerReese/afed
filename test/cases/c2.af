# Test Builtin Constants and Functions

v : 5 * pi * log(5,2)
sqrt(cos(v - 2*e) + 2) = 

# Try to apply log
sc:abs(sin(word)
	+ cos(pi/v))

log(sc, 2) - log(
sin(1 / 2)  ,
word
)=

word : ln(e + pi)

# Should produce no expr errors
notdefined =
mything: undef_func(word, 67) =

