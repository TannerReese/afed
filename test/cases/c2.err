(Line 18) EVAL_ERR_INCOMPLETE_CODE: Code Block doesn't have enough instructions
(Line 19) EVAL_ERR_INCOMPLETE_CODE: Code Block doesn't have enough instructions
