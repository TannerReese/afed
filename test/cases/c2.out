# Test Builtin Constants and Functions

v : 5 * pi * log(5,2)
sqrt(cos(v - 2*e) + 2) = 1.711364 

# Try to apply log
sc:abs(sin(word)
	+ cos(pi/v))

log(sc, 2) - log(
sin(1 / 2)  ,
word
)= 2.273173 

word : ln(e + pi)

# Should produce no expr errors
notdefined = ERR -5 
mything: undef_func(word, 67) = ERR -5 

